api/alembic/versions/362b687a9703_fire_starts_table_initial_data.py

```diff
@@ -0,0 +1,311 @@
+"""Fire starts table initial data
+
+Revision ID: 362b687a9703
+Revises: d072f1b6b6ee
+Create Date: 2022-03-10 13:29:05.603627
+
+"""
+from alembic import op
+from sqlalchemy.orm import Session
+
+from app.db.models.hfi_calc import FireCentre, HFIFireStarts
+
+
+# revision identifiers, used by Alembic.
+revision = '362b687a9703'
+down_revision = 'd072f1b6b6ee'
+branch_labels = None
+depends_on = None
+max_int = 2147483647
+
+
+def upgrade():
+    bind = op.get_bind()
+    s = Session(bind=bind)
+    for (fire_centre) in s.query(FireCentre):
+        if fire_centre.id == 1:
+            seed_kamloops_data(fire_centre, s)
+        else:
+            seed_standard_data(fire_centre, s)
+    s.commit()
+
+
+def seed_kamloops_data(kamloops_fire_centre: FireCentre, s: Session):
+    """ Seed IG to prep level data specific to kamloops (0 - 6+) """
+    objects = [
+        # 0 to 1
+        HFIFireStarts(fire_centre_id=kamloops_fire_centre.id,
```

Comment:
Halt. I have a more normalized suggestion. Also. We're stepping on each others toes. Let's chat before you proceed.